### PR TITLE
firefox-common-addons: fix whitelisting in ${RUNUSER}

### PR DESCRIPTION
**Pull request by glitsj16**
See #5495 for context.

### CODE REVIEW

File: etc/profile-a-l/firefox-common-addons.profile [hunk 1]
```diff
@@ -3,6 +3,10 @@
 include firefox-common-addons.local
 
 ignore whitelist ${RUNUSER}/*firefox*
+ignore whitelist ${RUNUSER}/psd/*firefox*
+# Add one of the following options to your firefox-common-addons.local if you enabled KeePassXC Plugin support.
+#ignore whitelist ${RUNUSER}/kpxc_server
+#ignore whitelist ${RUNUSER}/org.keepassxc.KeePassXC.BrowserServer
 ignore include whitelist-runuser-common.inc
 ignore private-cache
```
Comment (kmk3) [inline etc/profile-a-l/firefox-common-addons.profile:11]:
```suggestion
# Prevent whitelisting in ${RUNUSER}
ignore whitelist ${RUNUSER}/*firefox*
ignore whitelist ${RUNUSER}/kpxc_server
ignore whitelist ${RUNUSER}/org.keepassxc.KeePassXC.BrowserServer
ignore whitelist ${RUNUSER}/psd/*firefox*
ignore include whitelist-runuser-common.inc

ignore private-cache
```

It should be safe to always `ignore` the keepassxc `whitelist` commands.

Also, add a comment to make the intent clearer.

Comment (glitsj16) [reply etc/profile-a-l/firefox-common-addons.profile]: Indeed. Looks nicer this way!

**Review by kmk3 — APPROVED** [review]
LGTM.

**Comment by rusty-snake** [comment]
Not the last time.

**Comment by glitsj16** [comment]
> Not the last time.

@rusty-snake Can you clarify what you mean? Do you think there's something wrong with merging this?

**Comment by rusty-snake** [comment]
No, but it wasn't the first time fca.inc got patched after to fix nowhitelisting is RUNUSER and it will not be THW last time.

**Pull request merged**